packages/vat-data/package.json

```diff
@@ -32,6 +32,9 @@
   "ava": {
     "require": [
       "@endo/init/debug.js"
+    ],
+    "files": [
+      "test/**/test-*.js"
```

Comment:
I don't support this change but defer for the present.

Reply:
Without this packages.json change, would `yarn test` still test the test files now that they've been renamed? Or is there also an objection to renaming them to follow the convention we use elsewhere?

Reply:
The file rename depends on this config change. Here's what I want to see: https://github.com/Agoric/agoric-sdk/issues/8273

I concede it would be better to do all at once that deviate per package.

Reply:
Good. That's definitely better. I had no idea. Indeed better to do it all at once, because the dominant pattern is the natural example and attractor, no matter what some doc might say to the contrary. The current counts are

For endo
```
$ find . -name 'test-*.js' | wc
     298     298   15883
$ find . -name '*.test.js' | wc
      50      50    3569
```

For agoric-sdk
```
$ find . -name 'test-*.js' | wc
     391     391   21851
$ find . -name '*.test.js' | wc
      35      35    2344
```

Until at least a high supermajority of both are fixed to follow this convention, I won't be the last person to make this mistake.

That said, I definitely do like the new naming better!